tests/test_extract_metricvalues.py

```diff
@@ -125,7 +125,7 @@ def check_stdout(kind, n, last_line, contained, **kwargs):
                 self.assertIn(contained, result)
 
         # default call with no kwargs
-        contained = "{instrument: 'TestCam', detector: 25, visit: 54321, ...}"
+        contained = "{instrument: 'TestCam', detector: 25, visit: 54321, band: '447', physical_filter: '346'}"
```

Comment:
I think printing just the "basic" dimensions was actually a useful feature (though I don't know @parejkoj's thoughts on this). Is there no way to do that with the new system?

Reply:
Happy to switch it back, though the "..." will disappear.  Since it was hard to print the non-required values if and only if they were present before (but that's easy now), I've been converting stringifications that had the "..." (indicating that available values were not being printed) to print all available values.  But it doesn't have to be that way.

Reply:
I don't have a strong opinion on this, though having shorter dataIds can keep output from being too cluttered with keys that are irrelevant for a given use (but irrelevancy is often in the eye of the beholder).

Reply:
I have restored this stringification tested here to the concise version (now spelled `str(data_id.required)`), so the only change to the test is now dropping the `, ...` from the expected string.